### PR TITLE
Adding system definition API documentation

### PR DESCRIPTION
**Pull request by aaron-ni**
[x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?
* Adds system definition API documentation
* Updates sysdef_example.py to be a little clearer when excerpting from in documentation
* Adds the ability to run `tox -e docs` to generate the docs locally

### Why should this Pull Request be merged?
The system definition API is a major addition to the codebase and needs documentation. It's also easier to build the docs locally now (to preview doc changes) with the tox changes I made.

### What testing has been done?
Ran `tox -e flake8,docs,py38` on a system with VeriStand installed (all passed)

### CODE REVIEW

**Review by rtzoeller — APPROVED** [review]

File: docs/getting_started.rst [hunk 1]
```diff
@@ -7,9 +7,16 @@ Getting Started
 Features
 ========
 
-**niveristand** allows you to:
+**niveristand** has two major capabilities: system definition and real-time sequence scripting. It is recommended that you use an editor with code completion, such as `Visual Studio Code <https://code.visualstudio.com/docs/languages/python/>`_, to make it easier to browse and use this code.
```
Comment (rtzoeller) [inline docs/getting_started.rst:10]: You might get a technical writer to take a pass at this.

Comment (aaron-ni) [reply docs/getting_started.rst]: I've updated this based on the tech writer's comments.

**Comment by aaron-ni** [comment]
Waiting for Product Owner final review.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -7,9 +7,16 @@ Getting Started
 Features
 ========
 
-**niveristand** allows you to:
+**niveristand** has two major capabilities: system definition and real-time sequence scripting. NI recommends you use an editor with code completion, such as `Visual Studio Code <https://code.visualstudio.com/docs/languages/python/>`_, to make it easier to browse and use this code.
```
Comment (bdexni) [inline docs/getting_started.rst:10]: suggestion: "niveristand has two major capabilities: system definition scripting, and real-time sequence scripting and deployment." Even though we elaborate just below this in the real-time sequence section, I think it is worthwhile to call out that the python api specifically supports deploying rt sequences.

File: docs/sysdef_examples.rst [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+.. contents::
+   :depth: 4
+
+Basic Examples
+==============
+
+Creating a basic system definition file
+---------------------------------------
+A system definition can be created with either an initial target of either Windows or Linux.
```
Comment (bdexni) [inline docs/sysdef_examples.rst:15]: "A system definition can be created with a target operating system of Windows or Linux_x64."

File: docs/getting_started.rst [hunk 4]
```diff
@@ -7,9 +7,16 @@ Getting Started
...
 
-- Convert Python functions into NI VeriStand real-time (RT) sequences that are compatible with the NI VeriStand engine and Stimulus Profile Editor and save the converted Python functions to a file.
+Scripting system definition files
+---------------------------------
+You can script system definitions (.nivssdf) files for use in the NI VeriStand editor and deployable to the NI VeriStand engine.
```
Comment (bdexni) [inline docs/getting_started.rst:14]: "You can script system definition (.nivssdf) files for use in the NI VeriStand editor and deploy them to the NI VeriStand engine."

**Review by bdexni — CHANGES_REQUESTED** [review]

**Pull request merged**